en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md

```diff
@@ -8,22 +8,42 @@ The user has only the `apim:api_view` permission.
 Let's create a read only user and experience the Publisher portal in read only mode.
 
 Follow the instructions below to create a Read only user:
-1. Create role (e.g., `read_only`) in admin console.
-2. Create a user and assign the read_only role to the user.
-3. Assign the login permission to login to the Publisher portal via the admin console.
-4. Log in to the Admin Portal and navigate to **scope management** which is under **Settings**.
-5. Click **Add Scope Mappings**.
-   Enter the Role name as `read_only` (This is the role you assigned to the read only user).
-6. Click **Next**.
-7. Select the Custom scope assignments under the select permissions.
-   Check the `apim:api_view` permission under the Publisher.
+
+1. Create role (e.g., `read_only`) in admin console.Sign in to the management console (`https://<APIM_Host>:<APIM_Port>/carbon`) 
+   as the admin (default credentials are admin/admin). 
+   
+2. Click **Main**, and then click **Add** under **Users and Roles**.
+        
+      [![Add users and roles]({{base_path}}/assets/img/administer/add-users-and-roles.png)]({{base_path}}/assets/img/administer/add-users-and-roles.png)
+
+3. Click **Add New Role**.
+
+      ![Add new role]({{base_path}}/assets/img/administer/add-new-role.png)
+
+4.  Enter the name of the user role (e.g., `read_only`).
+
+      ![read-only-user-creation]({base_path}}/assets/img/learn/api-security/read_only_role.png)
+
+5. Create a user and assign the read_only role to the user.
+   
+6. Assign the login permission to login to the Publisher portal via the admin console.
+   
+7. Log in to the Admin Portal and navigate to **scope management** which is under **Settings**.
+   
+8. Click **Add Scope Mappings**.Enter the Role name as `read_only` (This is the role you assigned to the read only user).
+   
+9. Click **Next**.
+   
+10. Select the Custom scope assignments under the select permissions.Check the `apim:api_view` permission under the Publisher.
```

Comment:
The sentence "Check the...." looks like the action that corresponds to point 10. If so this need to be added in a separate line after leaving a blank link between point 10 and this sentence.